src/models/extractors.py

```diff
@@ -94,13 +94,24 @@ def __init__(self, config):
                 )
             ],
         )
+
+        # these values are backward compatible with `config.conv_padding = [1, 1]`
+        left_conv_padding = 2  # gets prepended to each line of feature matrix
```

Comment:
this cannot be hardcoded here, must be propagated from config

Reply:
this convolution is not over time -> there could be 0 padding

and 2 is there fore backward compatibility with existing models -> do we need that ?